easybuild/easyconfigs/m/mosdepth/mosdepth-0.2.4-foss-2018b.eb

```
@@ -31,7 +31,7 @@ dependencies = [
 extract_sources = True
 
 install_cmd = "cd %(builddir)s/hts-nim-*/ && nimble install --nimbleDir:%(installdir)s --verbose -y && "
-install_cmd += "cd ../mosdepth-*/ && sed -i 's/0\.2\.2/%(version)s/g' mosdepth.nimble && "
+install_cmd += r"cd ../mosdepth-*/ && sed -i 's/0\.2\.2/%(version)s/g' mosdepth.nimble && "
```

consistency across `mosdepth` easyconfigs
```suggestion
install_cmd += "cd ../mosdepth-*/ && "
install_cmd += r"sed -i 's/0\.2\.2/%(version)s/g' mosdepth.nimble && "
```